ietf/doc/templatetags/tests_ietf_filters.py

```diff
@@ -14,12 +14,15 @@
     ConflictReviewFactory,
     BofreqFactory,
     StatementFactory,
+    DocumentFactory,
```

Comment:
I think you should use WgDraftFactory rather than the bare DocumentFactory - it doesn't make sense to test this against non-ietf-stream documents.